File: KimTaeHwa/week_6/boj_2587.py
# 2587번 대표값2

arr = []
sum = 0


for i in range(5):
    arr.append(int(input()))
    
for i in range(5):
    sum += int(arr[i])

arr.sort()
    
print(int(sum/5))    
print(arr[2])

File: KimJiYu/week_6/boj_2587.java
package nestnet_algorithm_2024_2_beginner.KimJiYu.week_6;

import java.util.Arrays;
import java.util.Scanner;

/*
[2587] 대표값2
 */
public class boj_2587 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int[] arr = new int[5];
        int sum = 0;

        // 평균값
        for(int i = 0; i < 5; ++i){
            arr[i] = sc.nextInt();
            sum += arr[i];
        }
        System.out.println(sum/5);

        // 중앙값
        Arrays.sort(arr);
        System.out.println(arr[2]);

        sc.close();
    }
}
